python/grass/app/data.py

```diff
@@ -15,8 +15,12 @@
 import os
 import tempfile
 import getpass
+import subprocess
 import sys
 from shutil import copytree, ignore_patterns
+from pathlib import Path
+
+import grass.script as gs
```

Comment:
With this import and the other one of grass.app.data in the init inside lib/, I would like to make sure we are not introducing a circular import. (Ie, if grass.script requires that the lib init is imported, that requires grass.app, that requires grass.script). We have already multiple circular imports that Pylint 3 complains and have to be solved.

Reply:
That's indeed an issue to resolve. We don't have a plan for where should be the primary place where these are. I used lazy import of grass.app in grass.script.setup to avoid import-time circular dependencies. The lib/init code is not imported by the others, on the contrary, it should depend on the `grass` package and its subpackages (and it does).

Reply:
Thanks for the answer and the precision of for lib/init.
And the relationship of grass.app? When should it be imported and by who?

(At some point we'll need to draw a graph of dependencies and have transform it until it becomes a tree, not here)